cleanup_utils/aws_cleanup.py

```diff
@@ -129,7 +129,7 @@ def get_user_policy(self, username):
                 UserName=username,
                 PolicyName=DEFAULT_POLICY_NAME
             )
-        except botocore.exceptions.NoSuchEntityException:
+        except self.iam_client.exceptions.NoSuchEntityException:
```

Comment:
@giovannicimolin, didn't we discuss and decide to make a separate PR for this? If so, can you remove it?

Reply:
Yes, I missed it. I'll update the PR.

Reply:
Done